### ISSUE TITLE
The Play Button is off center

### ISSUE THREAD

**Issue by THEBIGTOMMY13DEV** [nimsandu/spicetify-bloom#365]
### Did you read the README?

- [X] I have read the README, the theme is installed/configured correctly and the "Troubleshooting" section contains no information related to my problem

### Do you have the latest versions of Spicetify, Spotify and Bloom?

- [X] I'm using the latest versions of Spicetify, Spotify and Bloom

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Is your problem definitely caused by the theme?

- [X] I have checked the app behavior without the theme and it is different

### Environment & Computer Info

```Markdown
- Spotify version:1.2.42.290.g242057a2
- Installed from: (Windows Store, Spotify site, Flatpak, etc)
- Spicetify version:v2.36.15
- Color scheme:Dark
- Extensions:none
- Custom apps:none
- Bloom installation method: (Spicetify Marketplace/PowerShell script/Bash script/Manual) Manual
```

### Description

I just install it and that's what it is

### Screenshots

![Uploading Screenshot 2024-07-24 130143.png…]()

**Comment by RazerGhost** [comment]
Having the same issue ☝️

**Comment by mdxwzl** [comment]
Same for me

**Comment by JeanxPereira** [comment]
Same

**Comment by fraudnome** [comment]
+

**Comment by FanciestBoi** [comment]
same issue

**Comment by JeanxPereira** [comment]
```css
.player-controls__buttons button {
  max-height: 32px /* var(--button-size) */;
  max-width: 32px /* var(--button-size) */;
  /* padding-left: 1rem;  */
  /* padding-right: 1rem; */
}
```
![image](https://github.com/user-attachments/assets/1ddef210-6aac-417b-9300-4f82b50040e0)

**Comment by Nimai-HK** [comment]
![image](https://github.com/user-attachments/assets/ccb4ed5e-4fe7-45f6-a996-17814081b024)
hovering over play button in dark fluent theme, the color is green ( not theme accent colour )

**Comment by im-h** [comment]
Forgot to send a message in here but i made a pull request with a fix to this [#367](https://github.com/nimsandu/spicetify-bloom/pull/367)

**Comment by sanoojes** [comment]
Added a fix !
